processor/src/main/java/org/mobilitydata/gtfsvalidator/processor/TableLoaderGenerator.java

```diff
@@ -165,10 +197,23 @@ private MethodSpec generateLoadMethod() {
 
         for (GtfsFieldDescriptor field : fileDescriptor.fields()) {
             method.addStatement(
-                    "final int $LColumnIndex = csvFile.getColumnIndex($L)",
-                    field.name(),
-                    fieldNameField(field.name()));
+                    "final int $L = csvFile.getColumnIndex($L)",
+                    fieldColumnIndex(field.name()), fieldNameField(field.name()));
+        }
+
+        // Several fields may reuse the same cache.
+        Set<String> cacheVars = new HashSet<>();
+        for (GtfsFieldDescriptor field : fileDescriptor.fields()) {
+            if (cachingEnabled(field)) {
+                String cacheVarName = fieldColumnCache(field);
+                if (cacheVars.add(cacheVarName)) {
+                    method.addStatement(
+                            "final $T<$T> $L = new $T<>()",
+                            FieldCache.class, TypeName.get(field.javaType()), cacheVarName, FieldCache.class);
+                }
+            }
         }
+
```

Comment:
NIT - extra blank line

Reply:
Yuck! I need to configure clang-format to format everything automatically. The formatting that you see is applied by IDEA.